### ISSUE TITLE
Base64 decode doesn't work

### ISSUE THREAD

**Issue by allanchau** [idearium/cli#20]
For some reason using node to decode strings doesn't seem to work.

I propose we update the command to use bash

```bash
# encode
echo XXXXX | base64

# decode
echo XXXXX | base64 -D
```

**Comment by allanchau** [comment]
In addition to this it should copy the output to the clipboard
```shell
$ c kc secret XXXXX
Copied XXXXX to the clipboard
$
```

**Comment by smebberson** [comment]
I was thinking the command should be updated to set and get the value from the actual YAML file...

**Comment by allanchau** [comment]
Yeah that would be good, a bit more complex though when you take into account the different environments.
